spec.emu

```diff
@@ -292,8 +292,15 @@ location: https://github.com/tc39/proposal-measure/
         1. Set _O_.[[Value]] to _roundedValue_.
         1. Set _O_.[[SignificantDigits]] to _significantDigits_.
         1. Set _O_.[[FractionDigits]] to _fractionDigits_.
-        1. If _unit_ is a String, set _O_.[[Unit]] to toLowercase(StringToCodePoints(_unit_)).
-        1. If _currency_ is a String, set _O_.[[Currency]] to toUppercase(StringToCodePoints(_unit_)).
+        1. If _unit_ is a String, then
+          1. If _unit_ is *""*, throw a *RangeError* exception.
+          1. If _unit_ contains a non-ASCII-alphanumeric character, throw a *RangeError* exception.
+          1. Set _O_.[[Unit]] to CodePointsToString(toLowercase(StringToCodePoints(_unit_))).
```

Comment:
The `toLowercase()` usage probably needs a similar Unicode reference as in https://tc39.es/ecma262/#sec-string.prototype.tolowercase. Or could we make this an actual call to `String.prototype.toLowerCase()`?

Same applies to _currency_ a few lines below.

Reply:
I think this should be resolved now. Can you take a look?